iosApp/iosApp/Localizable.xcstrings

```diff
@@ -89,6 +71,28 @@
         }
       }
     },
+    "%@ %@ %@" : {
+      "comment" : "VoiceOver text for the vehicle status on the trip details page,\nex 'train approaching Alewife' or 'bus now at Harvard'",
```

Comment:
suggestion (non-blocking): These seem tricky - maybe adding brackets around each piece in the comment so it is clear where one unit begins and another ends?